linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/http/RewriteHostHeader.scala

```diff
@@ -1,17 +1,33 @@
 package io.buoyant.linkerd.protocol.http
 
 import com.twitter.finagle.client.AddrMetadataExtraction.AddrMetadata
-import com.twitter.finagle.http.{Request, Response}
+import com.twitter.finagle.http.{Fields, Request, Response}
 import com.twitter.finagle.{Service, ServiceFactory, SimpleFilter, Stack}
 import io.buoyant.namer.Metadata
 
 object RewriteHostHeader {
 
+  val headers = Set(
+    Fields.Location,
+    "Refresh" // Not present in Fields... Probably because it's not widely sued but still...
+  )
+
   case class Filter(host: String) extends SimpleFilter[Request, Response] {
     def apply(req: Request, svc: Service[Request, Response]) = {
-      // TODO: switch between Host/:authority for HTTP1.1/2.0 when Finagle will support it
+      val originalHost = req.host
       req.host = host
       svc(req)
```

Comment:
I think this could use a commend explaining why we are replacing the host in the response headers.